prime-dotnet-webapi/Models/Documents/HealthAuthorityOrganizationAgreementDocument.cs

```diff
@@ -0,0 +1,8 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models
```

Comment:
Should this be in the namespace `Prime.Models.HealthAuthorities` instead?

Reply:
Could be, but to this point we have kept all documents in the Documents folder with no namespaces.

Reply:
Yeah while `SignedAgreementDocument` is still in `Prime.Models` namespace, it is under `Models\Agreements\SignedAgreementDocument.cs` on the file system